File: packages/insomnia-httpsnippet/test/fixtures/output/ocaml/cohttp/cookies.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "http://mockbin.com/har" in
let headers = Header.add (Header.init ()) "cookie" "foo=bar; bar=baz" in

Client.call ~headers `POST uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/jquery/cookies.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://mockbin.com/har",
  "method": "POST",
  "headers": {
    "cookie": "foo=bar; bar=baz"
  }
}

$.ajax(settings).done(function (response) {
  console.log(response);
});
